packages/swapper/src/swappers/thorchain/utils/bitcoin/utils/getThorTxData.ts

```diff
@@ -0,0 +1,96 @@
+import { btcChainId } from '@shapeshiftoss/caip'
+import { bitcoin } from '@shapeshiftoss/chain-adapters'
+import { HDWallet } from '@shapeshiftoss/hdwallet-core'
+import { Asset, BIP44Params, UtxoAccountType } from '@shapeshiftoss/types'
+
+import { SwapError, SwapErrorTypes } from '../../../../../api'
+import { bn, bnOrZero, fromBaseUnit, toBaseUnit } from '../../../../utils/bignumber'
+import { InboundResponse, ThorchainSwapperDeps } from '../../../types'
+import { getPriceRatio } from '../../getPriceRatio/getPriceRatio'
+import { makeSwapMemo } from '../../makeSwapMemo/makeSwapMemo'
+import { thorService } from '../../thorService'
+
+export const getThorTxInfo = async ({
```

Comment:
nicer to type `type GetBtcThorTxInfo = (args: GetBtcThorTxInfoArgs) => GetBtcThorTxInfoReturn`

Reply:
rename this to `getBTCThorTxInfo` and the existing other one to `getETHThorTxInfo`